tempest2/src/main/kotlin/app/cash/tempest2/Scan.kt

```diff
@@ -124,20 +129,59 @@ interface Scannable<K : Any, I : Any> {
       config.pageSize,
       config.consistentRead,
       config.filterExpression,
-      initialOffset
+      initialOffset,
+      config.workerId
     )
   }
 }
 
+/**
+ * By default, the Scan operation processes data sequentially. Amazon DynamoDB returns data to the
+ * application in 1 MB increments, and an application performs additional Scan operations to
+ * retrieve the next 1 MB of data.
+ *
+ * The larger the table or index being scanned, the more time the Scan takes to complete. In
+ * addition, a sequential Scan might not always be able to fully use the provisioned read throughput
+ * capacity: Even though DynamoDB distributes a large table's data across multiple physical
+ * partitions, a Scan operation can only read one partition at a time. For this reason, the
+ * throughput of a Scan is constrained by the maximum throughput of a single partition.
+ *
+ * To address these issues, the Scan operation can logically divide a table or secondary index into
+ * multiple segments, with multiple application workers scanning the segments in parallel. Each
+ * worker can be a thread (in programming languages that support multithreading) or an operating
+ * system process. To perform a parallel scan, each worker issues its own Scan request with an
```

Comment:
This is a lot of text to dump on this data class, and most of it applies to the the `scan` method and not the worker. If this was going to go anywhere I would expect it on an overload of `scan` that required the workedId parameter.

Reply:
Sounds good. That was just a copy-paste from the tempest1 implementation, but I can summarize and move it up. Thanks!